CONTRIBUTING.md

```diff
@@ -192,7 +192,7 @@ for f in ~/Downloads/packages-*.zip; do mkdir "$HOME/Downloads/$(basename -s .zi
 # we should now have all 5 build targets
 ls packages/core-bridge/releases/
 
-npx lerna version patch # or major|minor|etc, or leave out to be prompted. either way, you get a confirmation dialog.
+npx lerna version patch --force-publish=* # or major|minor|etc, or leave out to be prompted. either way, you get a confirmation dialog.
```

Comment:
This should be quoted to avoid shell expansion:
```suggestion
npx lerna version patch --force-publish='*' # or major|minor|etc, or leave out to be prompted. either way, you get a confirmation dialog.
```